minhphan156/Abode
# SJSU-1054_1: City Overview (Frontend)

**Story description:**
A city overview including city pictures, description of city, attractions in city, weather chart per month.

**What I did:**
_ Designed page for city overview.
_ Wired up action-redux workflow to React component.
_ Connected OpenWeatherMap API to redux.

**How to test:**
_ Go to localhost:3000/cities/0
_ Run the app. See if there are any errors caused by my code.
_ Look at the page using varying browser sizes to test responsiveness.
_ Look at my code. Let me know if there's ambiguity or incorrect usage of react, redux, etc..
_ Check if the story requirements are met.
